internationalization/supportedLanguages.ts

```diff
@@ -0,0 +1,31 @@
+export interface Language {
+  _key: string;
+  title: string;
+  id: string;
+  icon: string;
+  default?: boolean;
+}
+
+export const supportedLanguages: Language[] = [
+  {
+    _key: `${Date.now()}-${Math.random().toString(36).slice(2, 11)}`,
+    title: "English",
+    id: "en",
+    icon: "🇬🇧",
+    default: true,
```

Comment:
Can we have a separate value for default language? This avoids multiple defaults, and would be easier to use directly (avoid `.find()`)